Core/scripts/dirac-deploy-scripts.py

```diff
@@ -77,9 +77,24 @@
   args = ' "%s"' % '" "'.join( sys.argv[1:] )
 else:
   args = ''
-sys.exit( os.system('python "%s"%s' % ( DiracScript, args )  ) / 256 )
 """
 
+# Python interpreter location can be specified as an argument
+pythonLocation = "/usr/bin/env python"
+if len( sys.argv ) == 2:
+  pythonLocation = os.path.join( sys.argv[1], 'bin', 'python' )
+wrapperTemplate = wrapperTemplate.replace( '$PYTHONLOCATION$', pythonLocation )
+
+# On the newest MacOS the DYLD_LIBRARY_PATH variable is not passed to the shell of
+# the os.system() - bug or feature ?
```

Comment:
It is a feature. The "System Integrity Protection" prevents inheritance of some variables for child processes when "protected" executables are used.
https://developer.apple.com/library/content/documentation/Security/Conceptual/System_Integrity_Protection_Guide/RuntimeProtections/RuntimeProtections.html#//apple_ref/doc/uid/TP40016462-CH3-SW1